swarm/pss/pss.go

```diff
@@ -232,12 +232,12 @@ func (p *Pss) Start(srv *p2p.Server) error {
 			}
 		}
 	}()
-	log.Debug("Started pss", "public key", common.ToHex(crypto.FromECDSAPub(p.PublicKey())))
+	log.Info("Started Pss", "pubkey", common.ToHex(crypto.FromECDSAPub(p.PublicKey())))
```

Comment:
i think maybe:
`log.Info("Started Pss")`
then separately have:
log.Info("Loaded EC keys", "pubkey", common.ToHex(crypto.FromECDSAPub(p.PublicKey())), "secp256 compressed", common.ToHex(crypto.CompressPublicKey(p.PublicKey())))

with that you can also resolve this one: https://github.com/ethersphere/go-ethereum/issues/884